src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -179,7 +180,11 @@ public void waitForVideoToEnd(final long timeout) {
   }
 
   private void clickElement(final String selector) {
-    wait.forElementClickable(By.cssSelector(selector)).click();
+    Actions builder = new Actions(driver);
+    builder.contextClick(wait.forElementClickable(By.cssSelector(selector)))
+        .moveToElement(driver.findElement(By.cssSelector(selector)))
+        .click(wait.forElementClickable(By.cssSelector(selector)))
```

Comment:
What is the reason to wait for the same element if clickable? Is it the same element as in line [#184](https://github.com/Wikia/selenium-tests/pull/1815/files#diff-c54a3204718bd260b05fc037e1e12789R184)?